sympy/utilities/tests/test_lambdify.py

```diff
@@ -581,3 +582,9 @@ def test_issue_2790():
 def test_ITE():
     assert lambdify((x, y, z), ITE(x, y, z))(True, 5, 3) == 5
     assert lambdify((x, y, z), ITE(x, y, z))(False, 5, 3) == 3
+
+def test_10375():
+    import numpy as np
+    m = Min(x, y, z)
+    l = lambdify([x,y,z], m)
```

Comment:
I tried something like this : 
```
 if isinstance(expr,Min) and modules[1] == 'numpy':
         import numpy
         from numpy import min
         args = []
         args.append([numpy.array([args])])
```
and some related other ways.But still it is taking the arguments in the same way.